.github/workflows/deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Release to PyPI
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+ build-n-publish:
+    name: Build and publish release to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
```

Comment:
what is this >- ?

Reply:
It's a yaml string formatting thing. it says "what follows is all one string, replace the newlines with spaces"
https://symfony.com/doc/current/components/yaml/yaml_format.html